### ISSUE TITLE
Is there a way to modify how Upptime sends a notification to Discord? Right now the message it sends is quite weird and I was wondering if there was a way to change how it appeared.

### ISSUE THREAD

**Issue by Souvikns** [upptime/upptime#357]
Is there a way to modify how Upptime sends a notification to Discord? Right now the message it sends is quite weird and I was wondering if there was a way to change how it appeared. 

[Webhook Right Now](https://gyazo.com/4ff600c86ca87a0897bb01545f26da69)

_Originally posted by @Space-Turtle0 in https://github.com/upptime/upptime/discussions/206_

**Comment by Souvikns** [comment]
As of now [runtime](https://github.com/upptime/uptime-monitor) is using [simple discord text](https://github.com/upptime/uptime-monitor/blob/5d86107a9a548529856344aa61e19cf93e95953e/src/helpers/notifme.ts#L221) for discord notification.  By using embed we can make the notification look better. 

I was trying to design a basic notification 
```json
{
    "username": "Upptime Bot",
    "embeds": [
        {
            "title": "Upptime ⬆",
            "color": 3066993,
            "description": "🟩 google (https://google.com) is back up"
        }
    ]
}
```
![image](https://user-images.githubusercontent.com/41781438/117846919-840e3580-b29f-11eb-981a-fd1f4a88ffd6.png)

Write now the text that is being generated is `🟩 google (https://google.com) is back up`. For posting a single text this is fine, but if we use discord embed, then we need a couple of other fields that we can leverage for building a better-looking notification. We do need to generate custom colors for different types of notifications. 

> @AnandChowdhary would love to work on this issue. 😊

**Comment by FxllenCode** [comment]
Yeah, this should either be a feature or overhauled completely. Currently the Discord notifications don't work great:

I have almost 12k+ people who see these notifications and I get a fair amount of complaints that the notifications don't look good nor are functional. (I should be able to add a ping role!)

I'd like to see this implemented! :)

**Comment by Phol87** [comment]
❤️🔗

**Comment by Phol87** [comment]
@Phol87

**Comment by woeterman94** [comment]
Anyy update on this?

**Comment by mleczakm** [comment]
Is upptime even actively maintained, or should it be forked?
